### ISSUE TITLE
Cannot Mint Sigma Because Wallet is Locked

### ISSUE THREAD

**Issue by catchingknives** [zcoinofficial/zcoin#418]
When you try to mint Sigma with an encrypted wallet, you get the message
```
You cannot mint Sigma because Error: Wallet locked, unable to create transaction!
```
Corresponding log entry:
```
MintZerocoin() : Error: Wallet locked, unable to create transaction!
```

Please redirect user to password prompt here.

**Comment by riordant** [comment]
Fix added in https://github.com/zcoinofficial/zcoin/pull/427

**Comment by catchingknives** [comment]
Solved in #427
